tests/e2e/pageObjects/RegisterPage.js

```diff
@@ -1,36 +1,39 @@
 require("dotenv").config();
+
 const config=require("../../cucumber.conf.js")
+
 class RegisterPage {
+
   constructor() {
     this.baseUrl = config.tandoorURL;
     this.setupURL = this.baseUrl + "setup/";
-    this.username = `//input[@name="name"]`;
-    this.password = `#id_password`;
-    this.cPassword = `//input[@name="password_confirm"]`;
-    this.registerBtn = `//button[@class="btn btn-success"]`;
-    this.errorMsg = `//div[@id="div_id_password"]/div`;
-    this.succesMsg = `//div[@class="alert alert-success alert-dismissible fade show"]`;
+    this.usernameSelector = `//input[@name="name"]`;
+    this.passwordSelector = `#id_password`;
```

Comment:
for consistency if the name is available
```suggestion
    this.passwordSelector = '//input[@name="password"]';
```